jazzmin/templatetags/jazzmin.py

```diff
@@ -128,16 +128,15 @@ def get_jazzmin_settings(request: WSGIRequest) -> Dict:
     """
     settings = get_settings()
 
-    if hasattr(request, "current_app"):
-        admin_site = {x.name: x for x in all_sites}.get(request.current_app, "admin")
-        if not settings["site_title"]:
-            settings["site_title"] = admin_site.site_title
+    admin_site = {x.name: x for x in all_sites}.get("admin", {})
```

Comment:
if the default value for this `.get` is a dictionary, then we should probably not be using `getattr` on it.

Also, how comes you have removed the `request.current_app` key ?

It would be good to know what error you are receiving, as i think we might be able to find a more compatible solution

Reply:
You are right. Setting default value to `dict `and using `getattr `is confusing. We've used `getattr `since, if defined, `admin_site `should be instance of `AdminSite `which does not have a get method. It might be better to remove the default value to make it `None` when not available, since `getattr `can handle `None`.

We've removed the `request.current_app` since it is only defined when `admin `app is the current app. So it doesn't exists when browsing admindocs pages. We can keep it and set default to "admin" as before. We wanted to be sure if app name in admindoc is defined later on it won't pick up adminsdocs name.

Btw, we did not receive any errors, but the site_title showed up as None:
![image](https://user-images.githubusercontent.com/695823/166456199-9c367894-8163-4455-9da0-edffd884b1c5.png)

We have defined title via `site_header `in out custom admin class instead of jazzmin settings. If it's defined in jazzmin settings, this error is not replicated as it gets everything from `get_settings()` properly.

Reply:
Apologies for not getting this in sooner